### ISSUE TITLE
UI closes when switching tabs or window

### ISSUE THREAD

**Issue by marceljay** [polkadot-js/extension#400]
On Firefox 79, macOS Catalina:

When creating a new account from a new seed, the moment you switch tabs to paste the seed phrase, the UI closes and you have to restart the process. 
A user not paying attention could even restart the process, not noticing a new seed phrase, thinking they already copied it and thus creating a new account which they have **not** stored the seed phrase for. 

Proposed solution: When switching back to the window or tab, and opening the extension from the button, the last state should appear again, at least for some time (e.g. 240 seconds) to save the seed in e.g. a password manager.

**Comment by Tbaut** [comment]
Regarding the implementation, I'm thinking about context variable that is set to `{mnemonic, ttl: now() + 2min}` that would allow an automatic redirection. The one part that bothers me is the TTL reset. As long as there is no interaction, the variable with the mnemonic will stay in memory, potentially longer than 2min. We could eventually take advantage of [`chrome.alarm`](https://developer.chrome.com/extensions/alarms) to make sure the variable is effectively reset. What do you think?

**Comment by jacogr** [comment]
It is fine if it is active in the background page, it is not accessible. (So the timeout needs to be managed there)

**Comment by jak-pan** [comment]
Easy solution that other wallets use is to open a new browser tab when doing this. You don't lose state here. 

The proposed solution is probably not ideal, because:
1. you will be sending and storing mnemonic
2. when the timer expires which is probably not seen by the user, it will be a problem anyway. (Unless you explicitly show timer for the account). 

The question then is with this solution, how long do you want that timer to be? Since user might want to also write it down and can close the popup while reading a guide for example (this "bug" occured to some users when they were claiming and reading a guide).

I can also see a possibility of unwanted mnemonic storage persistence. Don't see direct problem on the first sight, but it should be considered.

**Comment by jacogr** [comment]
Indeed, the opening the account in a tab is imho, an acceptable workaround and one that is available to all users. Not perfect, but doesn't close on the user.

Longer-term would love to move all the vault views/management to only operate in a tab (lots more real-estate to play wih), but it has not had any focus.

**Comment by luyenheo** [comment]
@`
![unnamed](https://user-images.githubusercontent.com/115646489/225676909-a2c16227-7929-4d3e-867f-3fa0c629fce4.jpg)
`

**Comment by ap211unitech** [comment]
This issue appears to be outdated. However, feel free to open a new issue with updated details if you're still experiencing the problem.
